### PR TITLE
Release notes 0.8.1

### PR DESCRIPTION
**Pull request by essiequoi**
# What does this PR do?

Creates release notes for v0.8.1

### CODE REVIEW

File: changelog/2022-02-28.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+### Change Instance Type in Sessions
+
+This allows you to upgrade or downgrade the compute capability of the session nodes while keeping all of your work in progress untouched. The following operations are supported:
+
+- Change to a spot instance from an on-demand instance and vice versa
+- Change to a CPU instance from a GPU instance and vice versa
+    
+    <aside>
+    💡 Switching between CPUs and GPUs might require some additional package installation and may increase Session resume time
```
Comment (stanbiryukov) [inline changelog/2022-02-28.md:21]: Does this happen automatically or does the user need to do anything? Will existing user installed packages need to be reinstalled?

**Review by stanbiryukov — APPROVED** [review]
Looks great, just one comment on if users need to reinstall any packages they may have in their current session if they then switch to a GPU.

File: changelog/2022-02-28.md (hunk 1 above)
Comment (alexandercort) [reply changelog/2022-02-28.md:21]: @essiequoi has this been tested and validated? Any documentation you can share here?

Comment (essiequoi) [reply changelog/2022-02-28.md]: good q stan. @nmiculinic can you weigh in on this?

**Comment by essiequoi** [comment]
Removed docs on changing instance type until that feature is fully tested. FYI @stanbiryukov @alexandercort

File: docs/platform/changelog.md [hunk 2]
```diff
@@ -4,8 +4,24 @@ description: What's new and improved in Grid.
...
-**Archived release notes can be found on** [**this page**](https://github.com/gridai/gridai/discussions) **in Github.**&#x20;
+**CLI version: 0.8.0**
+
+Spring cleaning came early. This release features a lot of backend magic that improves overall stability and UX with Grid. 
+
+Upgrade your CLI to `v0.8.0` with `pip install lightning-grid --upgrade`  
+
+Find us in our [Slack Community](http://gridai-community.slack.com) to say hi and/or to express your thoughts/questions.
+
+### Notable Fixes and Enhancements
```
Comment (oojo12) [inline docs/platform/changelog.md:18]: Can we add some of the notable datastore improvements here? This is something I think the community would really like to see. I believe the notable ones are

1.  Improved upload speeds
2. Fix of the Aorpc error

File: changelog/2022-02-28.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+**CLI version: 0.8.0**
+
+Spring cleaning came early. This release features a lot of backend magic that improves overall stability and UX with Grid. 
+
+Upgrade your CLI to `v0.8.0` with `pip install lightning-grid --upgrade`  
+
+Find us in our [Slack Community](http://gridai-community.slack.com) to say hi and/or to express your thoughts/questions.
+
+### Notable Fixes and Enhancements
```
Comment (oojo12) [inline changelog/2022-02-28.md:11]: We probably also want to add the removal of unhelpful stacktrace error messages.

File: docs/platform/changelog.md (hunk 2 above)
Comment (oojo12) [reply docs/platform/changelog.md:18]: Would be great to have you @rlizzo comment on the notable datastore changes.

File: changelog/2022-02-28.md (hunk 3 above)
Comment (essiequoi) [reply changelog/2022-02-28.md:11]: this was not in the list that @ematta provided me. was this added to the LTS branch? and has it been tested in QA?

File: docs/platform/changelog.md [hunk 4]
```diff
@@ -4,8 +4,24 @@ description: What's new and improved in Grid.
 import Note from "@site/src/components/Note";
```
Comment (pritamsoni-hsr) [inline docs/platform/changelog.md:4]: we don't need this file.

can you replace its content to

```
### Don't edit, this file is autogenerated by `make build`
```

File: docs/platform/changelog.md (hunk 2 above)
Comment (essiequoi) [reply docs/platform/changelog.md:18]: Thanks, femi!

File: docs/platform/changelog.md (hunk 4 above)
Comment (essiequoi) [reply docs/platform/changelog.md]: done. thanks!

**Review by oojo12 — APPROVED** [review]

**Review by rlizzo — APPROVED** [review]
Datastore stuff lgtm

**Pull request merged**